src/views/Home.vue

```diff
@@ -49,16 +48,34 @@
 
 
 <script>
+import definition from '@/helpers/definition';
+
 export default {
   name:'home',
   data () {
     return {
       gender: '',
-      year: 0,
-      month: 0,
-      date: 0,
+      yearList: definition.year,
+      monthList: definition.month,
+      dateList:[1],
+      yearValue: '1990年 (平成2)',
+      monthValue:'1',
+      dateValue:'1',
     }
   },
+  methods: {
+    calculation: function() {
```

Comment:
下記のように省略できますので、使っていきましょう。
```suggestion
    calculation() {
```